src/main/java/com/onlinebookstore/model/Book.java

```diff
@@ -0,0 +1,32 @@
+package com.onlinebookstore.model;
+
+import com.onlinebookstore.dto.BookDto;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "books")
+public class Book extends BookDto {
```

Comment:
bad idea to extend `BookDto` here.

1. Classes `Book` and `BookDto` have different purposes.
2. Didn't use the inheritance

Leave just 
```
public class Book {
```

Reply:
Yes, strange idea. I don't understand how I wrote this. Already corrected. Thank you!